components/CommentsList/CommentsList.test.js

```diff
@@ -0,0 +1,27 @@
+import { render, screen } from "@testing-library/react";
+import CommentsList from "./CommentsList";
+
+const testComments = [
+  {
+    id: "1",
+    context: "sehr Lecker.",
+    time: "Tue Jun 08 2023",
+  },
+  {
+    id: "2",
+    context: "This restaurant is very picturesque.",
+    time: "Tue Jun 07 2023",
+  },
+];
+
+test("render every comment with key", () => {
+  render(<CommentsList comments={testComments} />);
+
+  const comment1 = screen.getByText(/sehr Lecker./i);
+  const comment2 = screen.getByText(/This restaurant is very picturesque./i);
+  const list = screen.getAllByRole("list");
```

Comment:
the test failed. maybe you want to find 3 "listitem"? I am not sure the test fails at line 26.

Reply:
Thanks Lenz.  You are right, the problem is from "listitem"  , the `<ul>` hat the  role of "list"